tutorials/SDF/car_demo.md

```diff
@@ -0,0 +1,520 @@
+# Building your own robot
+
+In this tutorial we will learn how to build our own robot in SDFormat. We will build a simple two wheeled robot.
+
+ You can find the finished world SDFormat of this tutorial [here](car_demo.sdf).
+
+## What is SDF ?
+
+[SDFormat](http://sdformat.org/) (Simulation Description Format), sometimes abbreviated as SDF, is an XML format that describes objects and environments for robot simulators, visualization, and control.
+
+## Building a world
+
+We will start by building a simple world and then build our robot in it. Open a new file and copy the following code to it.
+
+```xml
+<?xml version="1.0" ?>
+<sdf version="1.7">
+    <world name="car_world">
+        <physics name="1ms" type="ignored">
+            <max_step_size>0.001</max_step_size>
+            <real_time_factor>1.0</real_time_factor>
+        </physics>
```

Comment:
To make things simpler, we can remove this `<physics>` block.

Reply:
You mean because we don't have anything moving in our world so we can remove it?
I didn't explain anything related to the world I just refered to the world demo is fine or we need to do a just brief explanation

Reply:
This block is optional, so I was just thinking that we don't need to introduce it to users this early. But if you think it is important we can leave it in.

Reply:
Ok I will remove it. 
But do we need to add a brief description to world's tag or refering to the world tutorial is enough?

Reply:
I think just mentioning that the world will be explained in a future PR should be enough.